app/src/features/refusjon-omsorgspenger-arbeidsgiver/RefusjonOmsorgspengerArbeidsgiverForm.tsx

```diff
@@ -28,7 +28,8 @@ export const RefusjonOmsorgspengerArbeidsgiverSkjemaStateSchema = z.object({
     .array(
       z.object({
         dato: z.string().optional(),
-        antallTimer: z.number().optional(),
+        normalArbeidstid: z.number().optional(),
+        timerFravær: z.number().optional(),
```

Comment:
jeg renamet `antallTimer` til `timerFravær`, da det var litt mer klart nå som det var to timesummer som skulle defineres.